distribution/lib/Standard/Test/0.0.0-dev/src/Extensions.enso

```diff
@@ -41,9 +46,10 @@ Any.should_fail_with self matcher frames_to_skip=0 =
 
          example_should_fail_with =
             Examples.throw_error . should_fail_with Examples.My_Error
-Error.should_fail_with : Any -> Integer -> Test_Result
-Error.should_fail_with self matcher frames_to_skip=0 =
-    caught = self.catch
+Error.should_fail_with : Any -> Integer -> Boolean -> Test_Result
+Error.should_fail_with self matcher frames_to_skip=0 unwrap_errors=False =
+    unwrap_maybe error = if unwrap_errors then Error.unwrap error else error
+    caught = unwrap_maybe self.catch
```

Comment:
Shouldn't `unwrap_errors` default to `True`?

I imagine everywhere other than `Map_Error` tests, we actually want the unwrapping behaviour.

Reply:
Done